cfme/tests/containers/test_ssl_providers.py

```diff
@@ -49,9 +48,38 @@
 )
 
 
+@pytest.fixture(scope="module")
+def sync_ssl_certificate(provider, appliance):
+    """
+    fixture which sync SSL certificate between CFME and OCP
+    :param provider:  OCP provider object
+    :param appliance: CFME appliance object
+    """
```

Comment:
The correct docblock format is found here: https://pypi.python.org/pypi/sphinxcontrib-napoleon

Reply:
I hope that good now, please take a look at it.